static/app/components/replays/replayContext.tsx

```diff
@@ -193,6 +193,11 @@ type Props = {
 
   replay: ReplayReader | null;
 
+  /**
+   * Start the video as soon as it's ready
+   */
+  autoStart?: boolean;
```

Comment:
autoplay is really jarring when things are loading in, on my screen there's a bunch of flicker then it just starts moving. Also there's no auto-play to the next video, so it feels unbalanced. We should pick one mode, either full autoplay or none imo

relatedly, the clip player doesn't reserve enough space, causes significant repaint, when the selected replay switches :(